runtimes/huggingface/tests/test_tasks/test_japanese_mask_fill.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from mlserver.settings import ModelSettings, ModelParameters
+from mlserver_huggingface import HuggingFaceRuntime
+from mlserver_huggingface.codecs import HuggingfaceRequestCodec
+
+
+@pytest.fixture
+def bert_japanese_settings(request) -> ModelSettings:
+    if (not request.config.option.test_hg_tasks) or (
+        not request.config.option.test_ja_support
+    ):
+        pytest.skip()
+
+    return ModelSettings(
+        name="model",
+        implementation=HuggingFaceRuntime,
+        parameters=ModelParameters(
+            extra={
+                "task": "fill-mask",
+                "pretrained_model": "cl-tohoku/bert-base-japanese",
+                "pretrained_tokenizer": "cl-tohoku/bert-base-japanese",
+            }
+        ),
+    )
+
+
+@pytest.fixture
+async def current_runtime(bert_japanese_settings: ModelSettings) -> HuggingFaceRuntime:
+    runtime = HuggingFaceRuntime(bert_japanese_settings)
+    await runtime.load()
+    return runtime
+
+
+@pytest.fixture
+def japanese_text_request():
+    # Test sentence: Is the sky really [MASK]?
+    test_sentence = "実際に空が[MASK]のか？"
+    # [MASK] = visible
+    expected_output = "見える"
+
+    outputs = [expected_output]
+    reqs = HuggingfaceRequestCodec.encode_request(
+        {"inputs": [test_sentence]},
+        use_bytes=False,
+    )
+    return reqs, outputs
+
+
+async def test_infer(current_runtime, japanese_text_request):
```

Comment:
I think that this test function should include a parametrised (ie `pytest.mark.parametrize`) list of test cases in the form of [(`input_text`, `expected_response`),...] instead of hardcoding things  in the `japanese_test_request` fixture as it is not readable currently (ie separate out the expected response for it)

Reply:
Yeah that makes sense. I'll make this change.

Reply:
@sakoush I refactored the test to not make use of fixtures so we can use the `pytest.mark.parameterize` decorator. I have also moved the model config to a parameter. The single test case is documented above where we were using inline comments to describe the test case before. I believe this addresses your comment.